taxonomy/utils.py

```diff
@@ -129,7 +141,7 @@ def process_skills_data(product, skills, should_commit_to_db, product_type):
             }
             if should_commit_to_db:
                 update_skills_data(
-                    product[key_or_uuid], skill_external_id, confidence, skill_data, product_type
+                    product[key_or_uuid], skill_external_id, confidence, skill_data, product_type, **kwargs
```

Comment:
We should add the optional argument that we are anticipating in kwargs in the comment of the function.

Reply:
This needs to be done 😋

Reply:
Updated! previously updated the `update_skills_data` function docs. :)